File: code/java-spring/src/main/java/com/foodtruck/HelloController.java
package com.foodtruck;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

    @GetMapping("/")
    public HelloResponse hello() {
        return new HelloResponse("Hello, Food Truck Challenge!");
    }
}

File: code/dotnet-aspnet/HelloController.cs
using Microsoft.AspNetCore.Mvc;

namespace FoodTruck.API.Controllers;

[ApiController]
[Route("[controller]")]
public class HelloController : ControllerBase
{
    [HttpGet]
    public HelloResponse Get()
    {
        return new HelloResponse("Hello, Food Truck Challenge!");
    }
}
